### PR TITLE
Revert "EIP-778: fix type field"

### PR DESCRIPTION
**Pull request by sea51930**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-X.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 778 requires approval from one of (fjl@ethereum.org)

File: EIPS/eip-778.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 eip: 778
 title: Ethereum Node Records (ENR)
 author: Felix Lange <fjl@ethereum.org>
-type: Standards Track
+type: Standard Track
```
Comment (5chdn) [inline EIPS/eip-778.md]: I thought it's Standard**s** Track?

**Comment by Arachnid** [comment]
@sea51930 Why are you trying to revert this?

**Comment by fjl** [comment]
@Arachnid why did you merge it? ;)

**Pull request merged**